### PR TITLE
Add an example of workflow

### PR DESCRIPTION
**Pull request by xyxxxxx**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,5 @@
+# Jupyter notebook tmp directory
```
Comment (ppkube) [inline .gitignore:1]: please ignore `.DS_Store` files

File: workflow/automatic-workflow/workflowtemplates/100-tfdv.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+apiVersion: batch.tensorstack.dev/v1beta1
+kind: WorkflowTemplate
+metadata:
+  name: data-validation
+  labels:
```
Comment (ppkube) [inline workflow/automatic-workflow/workflowtemplates/100-tfdv.yaml:5]: are these labels recommended? I don't see them on other YAML files.

File: workflow/automatic-workflow/workflowtemplates/110-tft.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+apiVersion: batch.tensorstack.dev/v1beta1
+kind: WorkflowTemplate
+metadata:
+  name: data-transform
+  labels:
+    tensorstack.dev/component: taskman
```
Comment (ppkube) [inline workflow/automatic-workflow/workflowtemplates/110-tft.yaml:6]: pls ensure these labels are expected and consistent.

File: workflow/automatic-workflow/workflowtemplates/200-training.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+apiVersion: batch.tensorstack.dev/v1beta1
+kind: WorkflowTemplate
+metadata:
+  name: model-training
+  labels:
+    tensorstack.dev/component: taskman
```
Comment (ppkube) [inline workflow/automatic-workflow/workflowtemplates/200-training.yaml:6]: ditto

File: workflow/automatic-workflow/workflowtemplates/100-tfdv.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+      default: data
+    - name: RUNTIME_IMAGE
+      description: The docker image for runnning this task.
+      default: "registry.tensorstack.cn/t9k/tfx:prod-2.1.1"
+  type: SeqPod
+  seqPod:
+    steps:
+      - name: tf-data-validation
+        image: $(params.RUNTIME_IMAGE)
+        resources:
```
Comment (ppkube) [inline workflow/automatic-workflow/workflowtemplates/100-tfdv.yaml:31]: Are `resources` required? if not, for simplicity, we could simply skip them.

File: workflow/automatic-workflow/workflowtemplates/110-tft.yaml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+      default: data
+    - name: RUNTIME_IMAGE
+      description: The docker image for runnning this task.
+      default: "registry.tensorstack.cn/t9k/tfx:prod-2.1.1"
+  type: SeqPod
+  seqPod:
+    steps:
+      - name: tf-transform
+        image: $(params.RUNTIME_IMAGE)
+        resources:
```
Comment (ppkube) [inline workflow/automatic-workflow/workflowtemplates/110-tft.yaml:34]: ditto

File: workflow/automatic-workflow/workflowtemplates/200-training.yaml [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+      default: data
+    - name: RUNTIME_IMAGE
+      description: The docker image for runnning this task.
+      default: "registry.tensorstack.cn/t9k/tfx:prod-2.1.1"
+  type: SeqPod
+  seqPod:
+    steps:
+      - name: trainer
+        image: $(params.RUNTIME_IMAGE)
+        resources:
```
Comment (ppkube) [inline workflow/automatic-workflow/workflowtemplates/200-training.yaml:31]: ditto

File: workflow/automatic-workflow/workflowtemplates/210-tfma.yaml [hunk 8]
```diff
@@ -0,0 +1,43 @@
+apiVersion: batch.tensorstack.dev/v1beta1
+kind: WorkflowTemplate
+metadata:
+  name: model-analysis
+  labels:
+    tensorstack.dev/component: taskman
```
Comment (ppkube) [inline workflow/automatic-workflow/workflowtemplates/210-tfma.yaml:6]: ditto

File: workflow/automatic-workflow/workflowtemplates/210-tfma.yaml [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+      default: data
+    - name: RUNTIME_IMAGE
+      description: The docker image for runnning this task.
+      default: "registry.tensorstack.cn/t9k/tfx:prod-2.1.1"
+  type: SeqPod
+  seqPod:
+    steps:
+      - name: analyze-model
+        image: $(params.RUNTIME_IMAGE)
+        resources:
```
Comment (ppkube) [inline workflow/automatic-workflow/workflowtemplates/210-tfma.yaml:31]: ditto

File: workflow/automatic-workflow/workflowtemplates/220-confusion-matrix.yaml [hunk 10]
```diff
@@ -0,0 +1,38 @@
+apiVersion: batch.tensorstack.dev/v1beta1
+kind: WorkflowTemplate
+metadata:
+  name: confusion-matrix
+  labels:
+    tensorstack.dev/component: taskman
```
Comment (ppkube) [inline workflow/automatic-workflow/workflowtemplates/220-confusion-matrix.yaml:6]: ditto

File: workflow/automatic-workflow/workflowtemplates/220-confusion-matrix.yaml [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+      default: output
+    - name: RUNTIME_IMAGE
+      description: The docker image for runnning this task.
+      default: "registry.tensorstack.cn/t9k/tfx:prod-2.1.1"
+  type: SeqPod
+  seqPod:
+    steps:
+      - name: compute-confusion-matrix
+        image: $(params.RUNTIME_IMAGE)
+        resources:
```
Comment (ppkube) [inline workflow/automatic-workflow/workflowtemplates/220-confusion-matrix.yaml:28]: ditto

File: workflow/automatic-workflow/workflowtemplates/220-roc.yaml [hunk 12]
```diff
@@ -0,0 +1,38 @@
+apiVersion: batch.tensorstack.dev/v1beta1
+kind: WorkflowTemplate
+metadata:
+  name: roc
+  labels:
+    tensorstack.dev/component: taskman
```
Comment (ppkube) [inline workflow/automatic-workflow/workflowtemplates/220-roc.yaml:6]: ditto

File: workflow/automatic-workflow/workflowtemplates/220-roc.yaml [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+      default: output
+    - name: RUNTIME_IMAGE
+      description: The docker image for runnning this task.
+      default: "registry.tensorstack.cn/t9k/tfx:prod-2.1.1"
+  type: SeqPod
+  seqPod:
+    steps:
+      - name: compute-roc
+        image: $(params.RUNTIME_IMAGE)
+        resources:
```
Comment (ppkube) [inline workflow/automatic-workflow/workflowtemplates/220-roc.yaml:28]: ditto

File: workflow/automatic-workflow/workflowtemplates/999-taxi-tips-prediction.yaml [hunk 14]
```diff
@@ -0,0 +1,117 @@
+apiVersion: batch.tensorstack.dev/v1beta1
+kind: WorkflowTemplate
+metadata:
+  name: taxi-tips-prediction
+  labels:
+    tensorstack.dev/component: taskman
```
Comment (ppkube) [inline workflow/automatic-workflow/workflowtemplates/999-taxi-tips-prediction.yaml:6]: ditto

**Comment by ppkube** [comment]
Also, please remove `workflow/automatic-workflow/workflowtemplates/.DS_Store`

**Review by ppkube — CHANGES_REQUESTED** [review]
Please make sure they can run successfully.

File: workflow/automatic-workflow/workflowtemplates/100-tfdv.yaml (hunk 2 above)
Comment (xyxxxxx) [reply workflow/automatic-workflow/workflowtemplates/100-tfdv.yaml:5]: 我不知道这些 label 有什么用，作者 @adux6991  能不能回答一下

Comment (adux6991) [reply workflow/automatic-workflow/workflowtemplates/100-tfdv.yaml:5]: 没有用，可以删掉。是很早以前我们设计 artifact web 的时候做的例子，一直 copy paste 了

File: workflow/automatic-workflow/workflowtemplates/100-tfdv.yaml (hunk 5 above)
Comment (adux6991) [reply workflow/automatic-workflow/workflowtemplates/100-tfdv.yaml:31]: 在部署了 admission control 的 t9k 集群中，不加 `resources` 创建不了 pod

File: workflow/automatic-workflow/workflowtemplates/100-tfdv.yaml (hunk 2 above)
Comment (xyxxxxx) [reply workflow/automatic-workflow/workflowtemplates/100-tfdv.yaml:5]: 删除这些 labels 之后可以运行

File: .gitignore (hunk 1 above)
Comment (xyxxxxx) [reply .gitignore:1]: ignored

File: workflow/automatic-workflow/workflowtemplates/100-tfdv.yaml (hunk 2 above)
Comment (xyxxxxx) [reply workflow/automatic-workflow/workflowtemplates/100-tfdv.yaml]: deleted

File: workflow/automatic-workflow/workflowtemplates/110-tft.yaml (hunk 3 above)
Comment (xyxxxxx) [reply workflow/automatic-workflow/workflowtemplates/110-tft.yaml]: ditto

File: workflow/automatic-workflow/workflowtemplates/200-training.yaml (hunk 4 above)
Comment (xyxxxxx) [reply workflow/automatic-workflow/workflowtemplates/200-training.yaml]: ditto

File: workflow/automatic-workflow/workflowtemplates/100-tfdv.yaml (hunk 5 above)
Comment (xyxxxxx) [reply workflow/automatic-workflow/workflowtemplates/100-tfdv.yaml:27]: keep it?

File: workflow/automatic-workflow/workflowtemplates/210-tfma.yaml (hunk 8 above)
Comment (xyxxxxx) [reply workflow/automatic-workflow/workflowtemplates/210-tfma.yaml]: deleted

File: workflow/automatic-workflow/workflowtemplates/220-confusion-matrix.yaml (hunk 10 above)
Comment (xyxxxxx) [reply workflow/automatic-workflow/workflowtemplates/220-confusion-matrix.yaml]: deleted

File: workflow/automatic-workflow/workflowtemplates/220-roc.yaml (hunk 12 above)
Comment (xyxxxxx) [reply workflow/automatic-workflow/workflowtemplates/220-roc.yaml]: deleted

File: workflow/automatic-workflow/workflowtemplates/999-taxi-tips-prediction.yaml (hunk 14 above)
Comment (xyxxxxx) [reply workflow/automatic-workflow/workflowtemplates/999-taxi-tips-prediction.yaml]: deleted

**Review by ppkube — APPROVED** [review]
LGTM

**Pull request merged**